### DISCUSSION TITLE
Phaser 3 Sandbox Online Development Experience Enhancements on https://labs.phaser.io/

### DISCUSSION THREAD

**Discussion by ELO2026X** [photonstorm/phaser#6750] [Feature Requests]
**Proposal:**
Implement a dark mode for labs.phaser.io to enhance the development experience. https://labs.phaser.io/

**Background:**
After encountering a development issue on RoseBud.AI, https://play.rosebud.ai/games/765c4007-8190-4376-b08e-f1096ec354ea I transitioned to the Phaser 3 Sandbox here: https://labs.phaser.io/edit.html?src=src\physics\arcade\basic%20platform.js to run my source code due to its stability and asset integration capabilities. The seamless integration with assets stored on storage.googleapis.com and the ability to test code directly on labs.phaser.io have proven invaluable.

**Benefits of Enhanced Sandbox:**
By leveraging the Phaser 3 Sandbox, more users can experiment with and learn about Phaser 3 technology directly through the Phaser server. This user-friendly IDE provides a convenient platform for beginners to explore Phaser 3's capabilities.

**Call to Action:**
I urge the Phaser development team to consider implementing additional features within the sandbox to further facilitate user exploration and learning. The ability to showcase Phaser 3's technology through this platform can significantly benefit the community.

![Screenshot 2024-02-28 033901](https://github.com/phaserjs/phaser/assets/153048533/0a0abe6c-b5cc-4588-a1e3-ded871bbcf05)
![sadf4ad4s7f864adsf85](https://github.com/phaserjs/phaser/assets/153048533/eb046a66-c02d-44d0-b123-a9425f517173)

**Comment by photonstorm** [comment]
Thank you for the suggestion - our Labs site will be retired and replaced with an entirely new one in the coming month. It was only ever meant to be a stop-gap solution, but it ended up being in use for several years. However, it's time has come to an end and we'll replace it with our new examples site and integrated Phaser Editor.
